src/routes/usuario.routes.js

```diff
@@ -0,0 +1,25 @@
+const { Router } = require('express')
+const Usuario = require('../models/usuario')
+const Token = require('../tools/token')
+
+const router = Router()
+
+router.post('/login', async(req, res) => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Backend/pullRequest?prid=7127635)
```suggestion
router.post("/login", async(req, res) => {
```